report/assets/styles.css

```diff
@@ -561,6 +562,36 @@
   border-bottom: var(--score-icon-size) solid var(--color-fail);
 }
 
+.lh-audit--error .lh-audit__score-icon,
+.lh-metric--error .lh-metric__icon {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  width: calc(var(--score-icon-size) * 2.4);
+  height: calc(var(--score-icon-size) * 2.4);
+  background-position: center;
+  margin-right: 0;
+  border: none;
+}
+
+.lh-gauge__wrapper--fail .lh-gauge--error {
+  background-image: var(--error-icon-url);
+  background-repeat: no-repeat;
+  background-position: center;
+  top: var(--score-container-padding);
+}
+
+.lh-sticky-header .lh-gauge__wrapper--fail .lh-gauge--error {
+  height: calc(var(--score-icon-size) * 3.25);
+  background-position: center;
+  top: calc(var(--score-container-padding) / 2);
+}
```

Comment:
If we remove this rule, the "!" just gets smaller. I would prefer to do that and keep the code concise.

Reply:
Cool, just removed the rule. Like you mentioned the "!" gets smaller but it still looks good.